core/src/main/java/org/infinispan/commands/functional/WriteOnlyManyEntriesCommand.java

```diff
@@ -93,7 +96,8 @@ public Object perform(InvocationContext ctx) throws Throwable {
 
    @Override
    public boolean isReturnValueExpected() {
-      return false;
+      // Scattered cache always needs some response.
```

Comment:
In scattered cache it will return an array of InternalCacheValue (after modification), because originator cannot apply the function locally (it does not have the previous version).